ledger/acctonline.go

```diff
@@ -511,10 +587,14 @@ func (ao *onlineAccounts) lookupOnlineAccountData(rnd basics.Round, addr basics.
 		if err != nil {
 			return
 		}
+		paramsOffset, err = ao.roundParamsOffset(rnd)
+		if err != nil {
+			return
+		}
 
 		// TODO: ensure protocol version and totals are available after shrinking deltas
-		rewardsProto = config.Consensus[ao.versions[offset]]
-		rewardsLevel = ao.roundTotals[offset].RewardsLevel
```

Comment:
looks like `roundTotals` and `versions` are not used anymore, feel free to delete